internal/config/config.go

```diff
@@ -207,9 +206,15 @@ func Load(fileName string) (*Configuration, error) {
 			Mode:  log.OutputText,
 		},
 	}
-	_ = viper.ReadInConfig()
-	_ = viper.Unmarshal(config)
-	checkEnvVars(config)
+	ctx := context.Background()
```

Comment:
context should be passed as a parameter to SanitizeAdmin() instead of creating a new one here.